packages/pyolite-kernel/src/worker.ts

```diff
@@ -183,39 +198,36 @@ async function execute(content: any) {
   interpreter.display_pub.update_display_data_callback = updateDisplayDataCallback;
   interpreter.displayhook.publish_execution_result = publishExecutionResult;
 
-  const res = await interpreter.run(content.code);
-  console.log(res);
-  const reply_content: any = {};
+  await interpreter.run(content.code);
+  const results: any = {};
 
-  reply_content['payload'] = interpreter.payload_manager.read_payload();
+  results['payload'] = formatResult(interpreter.payload_manager.read_payload());
   interpreter.payload_manager.clear_payload();
 
   if (typeof interpreter._last_traceback === 'undefined') {
-    reply_content['status'] = 'ok';
-    // TODO: set reply_content['user_expressions']
+    results['status'] = 'ok';
+    // TODO: set results['user_expressions']
   } else {
-    console.log(interpreter._last_traceback);
     const last_traceback = formatResult(interpreter._last_traceback);
```

Comment:
It seems you will need to call `formatResult` on each element individually (`formatResult` does not seem to work recursively on your `_last_traceback` object).